component/block_cache/block_cache.go

```diff
@@ -288,6 +288,8 @@ func (bc *BlockCache) Configure(_ bool) error {
 	}
 
 	bc.tmpPath = common.ExpandPath(conf.TmpPath)
+	
+	// Handle cleanup-on-start configuration
 	bc.cleanupOnStart = conf.CleanupOnStart
```

Comment:
As clean-up-on-start is generic now in mount.go, code related to cleanup on start can be removed from both block-cache and file-cache.